isis/src/kaguya/apps/kaguyatc2isis/main.cpp

```diff
@@ -128,6 +127,125 @@ void IsisMain() {
+          }
+          else { // encoding == "N/A" so no compression
+            kern += PvlKeyword("NaifFrameCode", toString(-131359));
+          }
+        }
+      }
+      else { // swath == "Half"
+        if (productSetId == "TC_w_Level2B0") {
+          if (encoding == "DCT") {
+            kern += PvlKeyword("NaifFrameCode", toString(-131360));
+          }
+          else { // encoding == "N/A" so no compression
+            kern += PvlKeyword("NaifFrameCode", toString(-131361));
+          }
+        }
+        else if (productSetId == "TC_s_Level2B0") { 
+          if (encoding == "DCT") {
+            kern += PvlKeyword("NaifFrameCode", toString(-131362));
+          }
+          else { // encoding == "N/A" so no compression
+            kern += PvlKeyword("NaifFrameCode", toString(-131363));
+          }
+        }
+      }
+    }
+    if (instId == "TC2") {
+      if (swath == "Full") {
+        if (productSetId == "TC_w_Level2B0") {
+          if (encoding == "DCT") {
+            kern += PvlKeyword("NaifFrameCode", toString(-131372));
+          }
+          else { // encoding == "N/A" so no compression
+            kern += PvlKeyword("NaifFrameCode", toString(-131373));
+          }
+        }
+        else if (productSetId == "TC_s_Level2B0") { 
+          if (encoding == "DCT") {
+            kern += PvlKeyword("NaifFrameCode", toString(-131374));
+          }
+          else { // encoding == "N/A" so no compression
+            kern += PvlKeyword("NaifFrameCode", toString(-131375));
+          }
+        }
+      }
+      else if (swath == "Nominal") {
+        if (productSetId == "TC_w_Level2B0") {
+          if (encoding == "DCT") {
+            kern += PvlKeyword("NaifFrameCode", toString(-131376));
+          }
+          else { // encoding == "N/A" so no compression
+            kern += PvlKeyword("NaifFrameCode", toString(-131377));
+          }
+        }
+        else if (productSetId == "TC_s_Level2B0") { 
+          if (encoding == "DCT") {
+            kern += PvlKeyword("NaifFrameCode", toString(-131378));
+          }
+          else { // encoding == "N/A" so no compression
+            kern += PvlKeyword("NaifFrameCode", toString(-131379));
+          }
+        }
+      }
+      else { // swath == "Half"
+        if (productSetId == "TC_w_Level2B0") {
+          if (encoding == "DCT") {
+            kern += PvlKeyword("NaifFrameCode", toString(-131380));
+          }
+          else { // encoding == "N/A" so no compression
+            kern += PvlKeyword("NaifFrameCode", toString(-131381));
+          }
+        }
+        else if (productSetId == "TC_s_Level2B0") { 
+          if (encoding == "DCT") {
+            kern += PvlKeyword("NaifFrameCode", toString(-131382));
+          }
+          else { // encoding == "N/A" so no compression
+            kern += PvlKeyword("NaifFrameCode", toString(-131383));
+          }
+        }
+      }
+    }
+    */
```

Comment:
Was there a reason why this is here...?

Reply:
Added a comment for now because I wasn't sure it should be deleted. @scsides What do you think?